src/Lorentz/Contracts/BaseDAO/Management.hs

```diff
@@ -27,8 +27,7 @@ transferOwnership = do
   dip $ do
     dip $ stGetField #sAdmin
 
-    eq
-    if_ (do none; stSetField #sPendingOwner; push False) (push True)
+    if IsEq then (do none; stSetField #sPendingOwner; push False) else (push True)
```

Comment:
One of the reasons why I think using `IsEq` is better is because you can put this `do` statement into multiple lines which looks more natural to me. Though since this whole line is not too long, probably it's fine. Another reason is that you can remove brackets now. Brackets around `do` don't look nice, IMO. The same applies to other places below.